lib/aur-chroot

```diff
@@ -113,8 +113,9 @@ if ((prepare)); then
 fi
 
 if ((build)); then
-    sudo PKGDEST="${PKGDEST:-$PWD}" makechrootpkg -r "$directory" \
-         "${makechrootpkg_bindmounts_ro[@]}" "${makechrootpkg_args[@]}"
+    sudo --preserve-env=GNUPGHOME env PKGDEST="${PKGDEST:-$PWD}" \
+         makechrootpkg -r "$directory" "${makechrootpkg_bindmounts_ro[@]}" \
+         "${makechrootpkg_args[@]}"
```

Comment:
A third break is probably too much (I'm aware devtools adds a break for every argument in some places, which is quite strange)